pgtransaction/config.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+import psycopg2.errors
+
+
+def retry_exceptions():
+    """The default errors caught when retrying.
+
+    Note that these must be psycopg2 errors.
```

Comment:
Because currently the code catches the base database `Error` class and checks the `__cause__`. Technically they don't have to be psycopg2 exceptions, but they do have to be exceptions raised by the database engine